src/Kanvas/AccessControlList/Actions/Assign.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+namespace Kanvas\AccessControlList\Actions;
+
+use Bouncer;
+use Kanvas\Users\Models\Users;
+
+class Assign
+{
+    /**
+     * __construct
+     * @param Users $user
+     * @param string $role
+     * @return void
+     */
+    public function __construct(
+        public Users $user,
```

Comment:
@FredPeal need to send the company making sure the user you are assigning exist along side you

this action has more reason to exist since it has additional app logic